src/DefaultJsonResponseNegotiator.cs

```diff
@@ -12,7 +12,7 @@ public class DefaultJsonResponseNegotiator : IResponseNegotiator
     {
         public bool CanHandle(IList<MediaTypeHeaderValue> accept)
         {
-            return accept.Any(x => x.MediaType.IndexOf("json", StringComparison.OrdinalIgnoreCase) >= 0);
+            return accept.Any(x => x.MediaType.ToString().IndexOf("json", StringComparison.OrdinalIgnoreCase) >= 0);
```

Comment:
Don't think you need the ToString as it's implicit

Reply:
It is needed, or `.Value`